apps/portals/src/configurations/nf/synapseConfigs/datasets.ts

```diff
@@ -26,6 +32,15 @@ export const datasetCardConfiguration: CardConfiguration = {
       'yearProcessed',
     ],
   },
+  labelLinkConfig: [
+    {
+      isMarkdown: false,
+      matchColumnName: 'studyId',
+      URLColumnName: 'studyId',
+      baseURL: 'Explore/Datasets/DetailsPage',
```

Comment:
Shouldn't this link to the Studies page? Can you check a few of the "Datasets" details pages and verify that these links work?
```suggestion
      baseURL: 'Explore/Studies/DetailsPage',
```